extensions/sockets/src/test/java/org/mule/extension/socket/protocol/SafeProtocolTestCase.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.socket.protocol;
+
+import org.mule.extension.socket.SocketExtensionTestCase;
+import org.mule.tck.junit4.rule.DynamicPort;
+
+import org.junit.Rule;
+import org.junit.Test;
+
+public class SafeProtocolTestCase extends SocketExtensionTestCase
+{
+    @Rule
+    public DynamicPort dynamicPort = new DynamicPort("port1");
```

Comment:
why do you some times use a port1 rule and some others a tcp_port??? looks like you never use tcp and udp at the same time. Just have one single dynamic port in the base class and always use the same